An integration test fails when the most recent update does not have a request

One of the integration tests is [failing](https://ci.centos.org/job/bodhi-pipeline/job/PR-2891/1/execution/node/331/log/) today:

```
15:28:40 __________________________ test_updates_query_details __________________________
15:28:40 
15:28:40 bodhi_container = DockerContainer(image=bodhi-ci-integration-bodhi:latest, id=716354aebf48104ac29c6f0996b4cc695ec66d4f5e43284e713b19c356da4917)
15:28:40 db_container = DockerContainer(image=postgres:latest, id=5bf96ada80c6d02619bbcdf80bb6e11246e0205f3288040a1ee1f671d090490c)
15:28:40 greenwave_container = DockerContainer(image=bodhi-ci-integration-greenwave:latest, id=0b2fcfe0abc30f6297e2d9b4e0cd6809a94019d293b17b0151cee9a4e77e9115)
15:28:40 
15:28:40     def test_updates_query_details(bodhi_container, db_container, greenwave_container):
15:28:40         """Test getting an update's details with ``bodhi updates query``"""
15:28:40         # Fetch the last update from the DB
15:28:40         db_ip = db_container.get_IPv4s()[0]
15:28:40         query_update = (
15:28:40             "SELECT "
15:28:40             "  users.name as username, "
15:28:40             "  releases.long_name as release, "
15:28:40             "  updates.* "
15:28:40             "FROM updates "
15:28:40             "JOIN users ON updates.user_id = users.id "
15:28:40             "JOIN releases ON updates.release_id = releases.id "
15:28:40             "ORDER BY date_submitted DESC LIMIT 1"
15:28:40         )
15:28:40         query_comments = (
15:28:40             "SELECT u.name as username, c.timestamp, c.karma, c.text FROM comments c "
15:28:40             "JOIN users u ON c.user_id = u.id "
15:28:40             "WHERE update_id = %s ORDER BY c.timestamp"
15:28:40         )
15:28:40         query_karma = (
15:28:40             "SELECT SUM(comments.karma) as karma FROM comments "
15:28:40             "JOIN updates ON comments.update_id = updates.id "
15:28:40             "WHERE update_id = %s"
15:28:40         )
15:28:40         query_ct = (
15:28:40             "SELECT builds.type FROM builds "
15:28:40             "JOIN updates ON builds.update_id = updates.id "
15:28:40             "WHERE update_id = %s LIMIT 1"
15:28:40         )
15:28:40         query_builds = "SELECT nvr FROM builds WHERE update_id = %s ORDER BY nvr"
15:28:40         conn = psycopg2.connect("dbname=bodhi2 user=postgres host={}".format(db_ip))
15:28:40         with conn:
15:28:40             with conn.cursor() as curs:
15:28:40                 curs.execute(query_update)
15:28:40                 update = _db_record_to_munch(curs, curs.fetchone())
15:28:40                 update.comments = []
15:28:40                 curs.execute(query_comments, (update.id, ))
15:28:40                 for record in curs:
15:28:40                     update.comments.append(_db_record_to_munch(curs, record))
15:28:40                 curs.execute(query_karma, (update.id, ))
15:28:40                 update.karma = _db_record_to_munch(curs, curs.fetchone()).karma
15:28:40                 curs.execute(query_ct, (update.id, ))
15:28:40                 update.content_type = _db_record_to_munch(curs, curs.fetchone()).type
15:28:40                 curs.execute(query_builds, (update.id, ))
15:28:40                 update.builds = [r[0] for r in curs]
15:28:40         conn.close()
15:28:40         # Run the command
15:28:40         result = _run_cli(bodhi_container, ["updates", "query", "--updateid", update.alias])
15:28:40         assert result.exit_code == 0
15:28:40         assert "Update ID: {}".format(update.alias) in result.output
15:28:40         assert "Content Type: {}".format(update.content_type) in result.output
15:28:40         assert "Release: {}".format(update.release) in result.output
15:28:40         assert "Status: {}".format(update.status) in result.output
15:28:40         assert "Type: {}".format(update.type) in result.output
15:28:40         assert "Severity: {}".format(update.severity) in result.output
15:28:40         assert "Karma: {}".format(update.karma) in result.output
15:28:40         expected_autokarma = (
15:28:40             "Autokarma: {u.autokarma}  [{u.unstable_karma}, {u.stable_karma}]"
15:28:40         ).format(u=update)
15:28:40         assert expected_autokarma in result.output
15:28:40 >       assert "Request: {}".format(update.request) in result.output
15:28:40 E       AssertionError: assert 'Request: None' in '================================================================================\n     ansifilter-2.13-1.fc28\n======...e has been pushed to testing.\n\n  http://localhost:8080/updates/FEDORA-2019-7ae964cbc9\n\n1 updates found (1 shown)\n'
15:28:40 E        +  where 'Request: None' = <built-in method format of str object at 0x7f526bb63870>(None)
15:28:40 E        +    where <built-in method format of str object at 0x7f526bb63870> = 'Request: {}'.format
15:28:40 E        +    and   None = Munch({'username': 'filiperosset', 'release': 'Fedora 28', 'id': 129262, 'title': 'ansifilter-2.13-1.fc28', 'stable_ka...'This update has been pushed to testing.'})], 'karma': 0, 'content_type': 'rpm', 'builds': ['ansifilter-2.13-1.fc28']}).request
15:28:40 E        +  and   '================================================================================\n     ansifilter-2.13-1.fc28\n======...e has been pushed to testing.\n\n  http://localhost:8080/updates/FEDORA-2019-7ae964cbc9\n\n1 updates found (1 shown)\n' = Munch({'exit_code': 0, 'output': '================================================================================\n  ...has been pushed to testing.\n\n  http://localhost:8080/updates/FEDORA-2019-7ae964cbc9\n\n1 updates found (1 shown)\n'}).output
15:28:40 
15:28:40 devel/ci/integration/tests/test_bodhi_cli.py:237: AssertionError
```

The integration tests download the database dump from the Fedora infrastructure and use that as input data. We probably shouldn't do that because then our test data is variable which makes it difficult to reproduce problems (for example, I see this error in CentOS CI but not when I run locally).

There are two ways to fix this that I can think of:

0. Don't load a database dump from infra, and instead produce some repeatable test data.
1. Fix this specific test: when the request is ```None```, the line is simply omitted from the output instead of rendering ```Request: None```.

--- a/devel/ci/integration/tests/test_bodhi_cli.py
+++ b/devel/ci/integration/tests/test_bodhi_cli.py
@@ -1,4 +1,4 @@
-# Copyright © 2018 Red Hat, Inc.
+# Copyright © 2018-2019 Red Hat, Inc.
 #
 # This file is part of Bodhi.
 #
@@ -234,7 +234,9 @@ def test_updates_query_details(bodhi_container, db_container, greenwave_containe
         "Autokarma: {u.autokarma}  [{u.unstable_karma}, {u.stable_karma}]"
     ).format(u=update)
     assert expected_autokarma in result.output
-    assert "Request: {}".format(update.request) in result.output
+    # If the update doesn't have a request, the CLI does not render the Request: line.
+    if update.request:
+        assert "Request: {}".format(update.request) in result.output
     # Notes are formatted
     formatted_notes = list(itertools.chain(*[
         textwrap.wrap(line, width=66)

